smart-contracts/test/Lock/withdrawByAddress.js

```diff
@@ -0,0 +1,61 @@
+const BigNumber = require('bignumber.js')
+
+const deployLocks = require('../helpers/deployLocks')
+const shouldFail = require('../helpers/shouldFail')
+
+const unlockContract = artifacts.require('../Unlock.sol')
+const TestErc20Token = artifacts.require('TestErc20Token.sol')
+const getUnlockProxy = require('../helpers/proxy')
+
+let unlock, lock
+let token
+
+contract('Lock / withdrawByAddress', accounts => {
+  let owner = accounts[0]
+
+  before(async () => {
+    unlock = await getUnlockProxy(unlockContract)
```

Comment:
Didn't `getUnlockProxy` just get changed to `getProxy` ?

Reply:
Good catch, yea I need to rebase this